std/digest/package.d

```diff
@@ -1212,3 +1212,131 @@ if (isInputRange!R1 && isInputRange!R2 && !isInfinite!R1 && !isInfinite!R2 &&
         assert(!secureEqual(hex1, hex2));
     }
 }
+
+/**
+ * Used to convert hex text to byte range
+ * Params:
+ *  hex = hexdecimal encoded byte array
+ * Returns: byte range
+ * Example:
+ * ---
+ * ubyte[2] sba = "0xAA".fromHexString.staticArray!2;
```

Comment:
The length (2) isn't known at compile-time?

Reply:
no, it is n't

it's example if user wanna get digest with certain size, like hash or something

Reply:
What would that example look like in code?

Reply:
> What would that example look like in code?

u mean how to use it?
```
ubyte[32] hash = "0xafaf".fromHexString.staticArray!32;
ec.sign(hash)
```

```
ubyte[] data = "0xafaf".fromHexString.array;
shaOf(data)
```

Reply:
I'm resolving this to unblock merging.